packages/react/src/docs/customization.stories.mdx

```diff
@@ -0,0 +1,108 @@
+import { Canvas, Meta, Story, Source, ArgsTable } from '@storybook/addon-docs';
+
+<Meta title="Getting Started/Customization"/>
+
+# Customization
+
+At some point you might want to customize the design system's colors or the look and feel of some components. This is totally possible, and we will explain how in this documentation.
+But before we start, we will need to explain the concept of **design tokens**.
+
+
+## What are design tokens ?
+
+Design tokens are the fundamental variables defining the precise behavior and rendering of ui components.
+
+For example it could be:
+
+- The primary color of a text element
+- The standard spacing between two elements
+- The border radius of a button
+- ...
+
+Design tokens are the building blocks of a design system. They are the smallest pieces of a design system, and they are used to build components.
+
+## How to customize design tokens ?
+
+Cunningham makes possible to create a configuration file at the root of your project, which will be used to customize the design tokens.
+
+### Typescript
+
+If you prefer using Typescript, create a file named `cunningham.ts` at the root of your project with the following content:
+
+<Source dark={true} code={`
+// cunningham.ts
+export \default {
+  theme: {
+    colors: {
+      "primary-500": "purple",
+    },
+  },
+};`}></Source>
```

Comment:
As this typescript, we could take opportunity to show where to import token type ?

Reply:
Done 🥵